packages/manager-components/src/components/navigation/menus/action/action.component.tsx

```diff
@@ -26,17 +26,58 @@ export interface ActionMenuItem {
   download?: string;
   href?: string;
   target?: OdsHTMLAnchorElementTarget;
+  color?: ODS_THEME_COLOR_INTENT;
   onClick?: () => void;
   label: ReactI18NextChild | Iterable<ReactI18NextChild>;
 }
 
+export interface ActionMenuLayout {
+  isCompact?: boolean;
+  hasVerticalEllipsis?: boolean;
+}
+
 export interface ActionMenuProps {
   items: ActionMenuItem[];
+  /**
+   * @deprecated Use layout.isCompact instead
+   * */
   isCompact?: boolean;
+  layout?: ActionMenuLayout;
```

Comment:
Why create an object `layout` for 4 props in total ? It's not worth it in my opinion
And "vertical ellipsis" is not a layout, it's the icon used in the menu button. So it's a bit misleading.

I would suggest just adding an optional `menuIcon?: ODS_ICON_NAME` and inside the `ActionMenu` component :
```
name: menuIcon || isCompact ? ODS_ICON_NAME.ELLIPSIS : ODS_ICON_NAME.ARROW_DOWN_CONCEPT
```

Reply:
it's fixed !